### DISCUSSION TITLE
How to use helm to install airflow on k8s offline。

### DISCUSSION THREAD

**Discussion by jiangjian0920** [apache/airflow#27127] [Q&A]
I didn't find a clear documentation

k8s version：v1.18.19
docker：19.3.15

**Comment by potiuk** [comment]
You have not found it, because there is none. There are no ready-recipes for that and you are mostly alone with that and have to solve it yourself.

You could also phrase your question in the form "How to install ANY Helm Chart offline". This is not specific to airflow, so you should ask it in the Helm issue tracking not in Airflow's one. Here is similar issue there https://github.com/helm/helm/issues/8347 and as you can see, they are also not able to answer it with 100% certaintly - but if you want to try - take airlfow chart and follow the generic advices they provided.

Basically - it's always mnual effort if you want to do it and usually that involve you customising the chart and solving some problems withit - dependent charts, images etc.

From what I ve hreard some of the users of our paid multiple thousands of dollars for licences for some commercial products that provided such fully -offline automated conversion of their installations (i believe those customers had very high security requirments so they were also willing to pay for that big bucks. 

My suggestion is you either attempt to do it yourself or find such a company and pay them to do it.
